src/applications/mhv-medical-records/components/RecordList/AllergyListItem.jsx

```diff
@@ -4,152 +4,149 @@ import { Link } from 'react-router-dom';
 import ItemList from '../shared/ItemList';
 import { sendDataDogAction } from '../../util/helpers';
 
-const AllergyListItem = props => {
-  const { record } = props;
-  return (
-    <va-card
-      background
-      class="record-list-item vads-u-padding-y--2p5 vads-u-margin-bottom--2p5 vads-u-padding-x--3 left-align-print margin-zero-print"
-      data-testid="record-list-item"
-    >
-      {/* web view header */}
-      <div className="vads-u-font-weight--bold vads-u-margin-bottom--0p5">
-        <Link
-          to={`/allergies/${record.id}`}
-          data-dd-privacy="mask"
-          data-dd-action-name="[allergy list - name Link]"
-          data-testid={`allergy-link-${record.id}`}
-          className="no-print"
-          onClick={() => {
-            sendDataDogAction('Allergies Detail Link');
-          }}
-        >
-          {record.name}{' '}
-          <span
-            className="sr-only"
-            data-dd-privacy="mask"
-            data-dd-action-name="[allergy list - date]"
-          >
-            {`on ${record.date}`}
-          </span>
-        </Link>
-      </div>
-
-      {/* print view header */}
-      <h2
-        className="print-only vads-u-margin-bottom--1 vads-u-margin-top--0"
-        aria-hidden="true"
+const AllergyListItem = ({ record }) => (
+  <va-card
+    background
+    class="record-list-item vads-u-padding-y--2p5 vads-u-margin-bottom--2p5 vads-u-padding-x--3 left-align-print margin-zero-print"
+    data-testid="record-list-item"
+  >
+    {/* web view header */}
+    <div className="vads-u-font-weight--bold vads-u-margin-bottom--0p5">
+      <Link
+        to={`/allergies/${record.id}`}
         data-dd-privacy="mask"
-        data-dd-action-name="[allergy list - name - Print]"
+        data-dd-action-name="[allergy list - name Link]"
+        data-testid={`allergy-link-${record.id}`}
+        className="no-print"
+        onClick={() => {
+          sendDataDogAction('Allergies Detail Link');
+        }}
       >
         {record.name}
```

Comment:
Does removing the space in `{record.name}{' '}` still appropriately add a space after the record name? The next tag is an inline `span`.